models/base/sources.yml

```diff
@@ -13,6 +13,35 @@ sources:
           - name: emission_time
           - name: event
 
+  - name: dim_user_pii
```

Comment:
I think the way this is set up you're saying there's an existing table in ClickHouse called `dim_user_pii.user_pii` when really this should be another entry in `models/base/schema.yml`.

Reply:
corrected